### PR TITLE
FE - Monitor details revamp

### PR DESCRIPTION
**Pull request by danielcj2**
- [x] Restructured page
- [x] Removed unused files 
- [x] Updated UI  
- [x] Implemented new charts
- [x] Added interactivity to bar charts

Light mode:
image
image
image

Dark mode:
image
image
image

### CODE REVIEW

**Comment by gorkem-bwl** [comment]
Damn so good Daniel!

File: Client/src/Pages/Monitors/Details/Charts/index.jsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  const getColorRange = (uptime) => {
+    return uptime > 80
+      ? { main: theme.palette.success.main, light: theme.palette.success.light }
+      : uptime > 50
+      ? { main: theme.palette.warning.main, light: theme.palette.warning.light }
+      : { main: theme.palette.error.text, light: theme.palette.error.light };
+  };
+
+  // TODO - REMOVE THIS LATER
+  let reversedData = [...data].reverse();
```
Comment (ajhollid) [inline Client/src/Pages/Monitors/Details/Charts/index.jsx:58]: Definitely TODO :joy:

File: Client/src/Pages/Monitors/Details/Charts/index.jsx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+        <Bar
+          dataKey="totalChecks"
+          maxBarSize={7}
+          background={{ fill: "transparent" }}
+        >
+          {reversedData.map((entry, index) => {
+            let { main, light } = getColorRange(entry.uptimePercentage);
+            return (
+              <Cell
+                key={`cell-${index}`}
```
Comment (ajhollid) [inline Client/src/Pages/Monitors/Details/Charts/index.jsx:101]: We should use a unique identifier here, adding the `cell` prefix doesnt bamboozle react :joy: Is there any unique data here?  I can't remember :joy:

File: Client/src/Pages/Monitors/Details/Charts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+          }
+        />
+        <Bar
+          dataKey="totalIncidents"
+          maxBarSize={7}
+          background={{ fill: "transparent" }}
+        >
+          {reversedData.map((entry, index) => (
+            <Cell
+              key={`cell-${index}`}
```
Comment (ajhollid) [inline Client/src/Pages/Monitors/Details/Charts/index.jsx:174]: Same for this one

**Review by ajhollid — APPROVED** [review]

**Pull request merged**